Algorithm.Python/BasicTemplateCryptoFutureAlgorithm.py

```diff
@@ -21,126 +21,126 @@ class BasicTemplateCryptoFutureAlgorithm(QCAlgorithm):
     # Initialise the data and resolution required, as well as the cash and start-end dates for your algorithm. All algorithms must initialized.
     # </summary>
 
-    def Initialize(self):
-        self.SetStartDate(2022, 12, 13)
-        self.SetEndDate(2022, 12, 13)
+    def initialize(self):
+        self.set_start_date(2022, 12, 13)
+        self.set_end_date(2022, 12, 13)
 
-        self.SetTimeZone(TimeZones.Utc)
+        self.set_time_zone(TimeZones.Utc)
 
         try:
-            self.SetBrokerageModel(BrokerageName.BinanceFutures, AccountType.Cash)
+            self.set_brokerage_model(BrokerageName.BINANCE_FUTURES, AccountType.CASH)
         except:
             # expected, we don't allow cash account type
             pass
 
-        self.SetBrokerageModel(BrokerageName.BinanceFutures, AccountType.Margin)
+        self.set_brokerage_model(BrokerageName.BINANCE_FUTURES, AccountType.MARGIN)
 
-        self.btcUsd = self.AddCryptoFuture("BTCUSD")
-        self.adaUsdt = self.AddCryptoFuture("ADAUSDT")
+        self.btc_usd = self.add_crypto_future("BTCUSD")
+        self.ada_usdt = self.add_crypto_future("ADAUSDT")
 
-        self.fast = self.EMA(self.btcUsd.Symbol, 30, Resolution.Minute)
-        self.slow = self.EMA(self.btcUsd.Symbol, 60, Resolution.Minute)
+        self.fast = self.EMA(self.btc_usd.symbol, 30, Resolution.MINUTE)
+        self.slow = self.EMA(self.btc_usd.symbol, 60, Resolution.MINUTE)
 
-        self.interestPerSymbol = {self.btcUsd.Symbol: 0, self.adaUsdt.Symbol: 0}
+        self.interest_per_symbol = {self.btc_usd.symbol: 0, self.ada_usdt.symbol: 0}
 
-        self.SetCash(1000000)
+        self.set_cash(1000000)
 
         # the amount of BTC we need to hold to trade 'BTCUSD'
-        self.btcUsd.BaseCurrency.SetAmount(0.005)
+        self.btc_usd.base_currency.set_amount(0.005)
         # the amount of USDT we need to hold to trade 'ADAUSDT'
-        self.adaUsdt.QuoteCurrency.SetAmount(200)
+        self.ada_usdt.quote_currency.set_amount(200)
 
     # <summary>
     # OnData event is the primary entry point for your algorithm. Each new data point will be pumped in here.
     # </summary>
     # <param name="data">Slice object keyed by symbol containing the stock data</param>
-    def OnData(self, slice):
-        interestRates = slice.Get(MarginInterestRate);
+    def on_data(self, slice):
+        interestRates = slice.Get(MarginInterestRate)
```

Comment:
`slice.get(<type>)` working when Pythonnet 2.0.33 is public

Reply:
not yet solved